diff --git a/out/Attribute.lua b/out/Attribute.lua
deleted file mode 100644
index 3ebec83e86608f0cdfe5c0be82394fe3d70e6961..0000000000000000000000000000000000000000
diff --git a/out/init.lua b/out/init.lua
index e3a4f19aca6b4c1ef33d1e77760ef652bca21e5d..da97f208a87f51e88032dfeb98f7bc92d127b859 100644
--- a/out/init.lua
+++ b/out/init.lua	
@@ -64,7 +64,6 @@ end
 local Signal = require(iconModule.Packages.GoodSignal)
 local Janitor = require(iconModule.Packages.Janitor)
 local Utility = require(iconModule.Utility)
-local Attribute = require(iconModule.Attribute)
 local Themes = require(iconModule.Features.Themes)
 local Gamepad = require(iconModule.Features.Gamepad)
 local Overflow = require(iconModule.Features.Overflow)
