diff --git a/node_modules/@rbxts/wcs/out/source/client.luau b/node_modules/@rbxts/wcs/out/source/client.luau
index 4499ad4..abcdef0 100644
--- a/node_modules/@rbxts/wcs/out/source/client.luau
+++ b/node_modules/@rbxts/wcs/out/source/client.luau
@@ -250,7 +250,6 @@
			end
			return returnedValue
		end)
-       logMessage("Started Client successfully")
	end

diff --git a/node_modules/@rbxts/wcs/out/source/server.luau b/node_modules/@rbxts/wcs/out/source/server.luau
index f8cf6f5..1234567 100644
--- a/node_modules/@rbxts/wcs/out/source/server.luau
+++ b/node_modules/@rbxts/wcs/out/source/server.luau
@@ -304,7 +304,6 @@
		self._internal_isActive = true
-		logMessage("Started Server successfully")
	end
